src/System.Reflection.DispatchProxy/src/System.Reflection.DispatchProxy.csproj

```diff
@@ -11,15 +11,25 @@
     <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.6.0</AssemblyVersion>
     <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.7.0</AssemblyVersion>
     <PackageValidationBaseLineVersion>4.7.1</PackageValidationBaseLineVersion>
+    <IsPlaceholderTargetFramework Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</IsPlaceholderTargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
```

Comment:
I would rather set the default to true and opt-out for the TFMs that shouldn't use default items. That's more line with customer projects. From the list of TFMs above:
- netcoreapp3.1: placeholder TFM, already sets `EnableDefaultItems=false` in `placeholderpackaging.targets`. Nothing to do.
- netstandard2.1: type-forwards file only. `EnableDefaultItems` should be false.
- netstandard2.0: Needs DispatchProxy.cs only.
- net462: Needs everything except the type forwards file.

If you name `DispatchProxyGenerator.cs` to `DispatchProxyGenerator.netframework.cs` and `IgnoreAccessChecksToAttributeBuilder.cs` to `IgnoreAccessChecksToAttributeBuilder.netframework.cs`, these files will automatically get excluded on other TFMs: https://github.com/dotnet/maintenance-packages/blob/d4261b1112437567ca4841e0c251a0ec914e7213/Directory.Build.targets#L12

I'm not sure if that overall approach is more readable but I at least wanted to mention it.

Reply:
> If you name DispatchProxyGenerator.cs to DispatchProxyGenerator.netframework.cs and IgnoreAccessChecksToAttributeBuilder.cs to IgnoreAccessChecksToAttributeBuilder.netframework.cs, these files will automatically get excluded on other TFMs:

🤯

Reply:
> I'm not sure if that overall approach is more readable but I at least wanted to mention it.

It's readable and more obvious, but way more verbose.

Reply:
> It's readable and more obvious, but way more verbose.

Actually, nevermind, I refactored that even more and was able to come up with a very simple item group.